pyproject.toml

```diff
@@ -7,6 +7,14 @@ requires = [
 
 build-backend = "setuptools.build_meta"
 
+[tool.yapf]
```

Comment:
IMO, black with default configuration would have been sufficient. But of course I'm fine with everything that improves readability.

Reply:
I like the results of the proposed setup better than those produced by `black` because it is closer to the style used for most of the existing odxtools codebase. That said, discussing what the best style is is the main point of this PR. (and also, that we can avoid having to do such discussions again in the future ;))